cpp/src/IceSSL/SChannelTransceiverI.cpp

```diff
@@ -746,6 +746,18 @@ SChannel::TransceiverI::initialize(IceInternal::Buffer& readBuffer, IceInternal:
         throw SecurityException(__FILE__, __LINE__, "IceSSL: error reading cipher info:\n" + secStatusToString(err));
     }
 
+    try
```

Comment:
We call twice getInfo, once for verifyPeerCertName and another one for verifyPeer, it would be best to call it only once.